app/controllers/users/registrations_controller.rb

```diff
@@ -0,0 +1,26 @@
+class Users::RegistrationsController < Devise::RegistrationsController
+  respond_to :json
+
+  def create
+    user = User.create(user_params)
+    respond_with(user, _opts = {})
+  end
+
+  private
+
+  def user_params
+    params.require(:user).permit(:email, :password, :login, :role_id)
+  end
+
+  def respond_with(user, _opts = {})
```

Comment:
seems like argument `_opts` is not in use and could be removed

Reply:
Done